src/main/java/edu/ucsb/cs156/frontiers/controllers/RosterStudentsController.java

```diff
@@ -314,13 +314,24 @@ public RosterStudent updateRosterStudent(
     @PreAuthorize("hasRole('ROLE_INSTRUCTOR')")
     @DeleteMapping("/delete")
     @Transactional
-    public ResponseEntity<String> deleteRosterStudent(@Parameter(name = "id") @RequestParam Long id) throws EntityNotFoundException{
+    public ResponseEntity<String> deleteRosterStudent(@Parameter(name = "id") @RequestParam Long id) throws EntityNotFoundException {
         RosterStudent rosterStudent = rosterStudentRepository.findById(id)
                 .orElseThrow(() -> new EntityNotFoundException(RosterStudent.class, id));
         Course course = rosterStudent.getCourse();
+        
+        // Try to remove the student from the organization if they have a GitHub login
+        if (rosterStudent.getGithubLogin() != null && course.getOrgName() != null && course.getInstallationId() != null) {
+            try {
+                organizationMemberService.removeOrganizationMember(rosterStudent);
+            } catch (Exception e) {
+                log.error("Error removing student from organization: {}", e.getMessage());
+                // Continue with deletion even if organization removal fails
+            }
+        }
+        
         course.getRosterStudents().remove(rosterStudent);
         rosterStudentRepository.delete(rosterStudent);
         courseRepository.save(course);
-        return ResponseEntity.ok("Successfully deleted roster student and removed him/her from the course list");
+        return ResponseEntity.ok("Successfully deleted roster student and removed him/her from the course list and organization");
```

Comment:
Can we keep track of whether the student was, or was not, succesfully removed from the organization, and then customize the ResponseEntity message appropriately?   I'd like the options to be:

* Successfully deleted roster student and removed him/her from the course list and organization
* Successfully deleted roster student but there was an error removing them from the course organization: (e.getMessage() contents go here)